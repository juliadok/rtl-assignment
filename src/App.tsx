import React, { useEffect, useState } from 'react';
import { Alert } from 'react-bootstrap';
import Header from './components/Header';
import Articles from './components/Articles';

import './App.css';

function App(): React.ReactElement {
  const [error, setError] = useState<Error>();
  const [isLoading, setLoading] = useState(true);
  const [articles, setArticles] = useState<Articles>();

  useEffect((): void => {
    (async (): Promise<void> => {
      try {
        const response = await fetch(`${process.env.PUBLIC_URL}/bundle-api.json`);
        const data = await response.json();
        setArticles(data);
      } catch (e) {
        setError(e);
      } finally {
        setLoading(false);
      }
    })();
  }, []);

  if (isLoading) {
    return <Alert variant="primary">Loading...</Alert>;
  }

  if (error) {
    return <Alert variant="danger">{error.message}</Alert>;
  }

  if (!articles) {
    return <Alert variant="danger">No articles</Alert>;
  }

  return (
    <>
      <Header />
      <Articles articles={articles} />
    </>
  );
}

export default App;
