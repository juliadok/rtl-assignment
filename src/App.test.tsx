import React from 'react';
import { render } from 'react-dom';
import { act } from 'react-dom/test-utils';

import articles from '../public/bundle-api.json';
import App from './App';

describe('App', () => {
  let container: HTMLElement;

  beforeEach(async () => {
    container = document.createElement('div');
  });

  it('should render articles', async () => {
    jest.spyOn(global, 'fetch').mockResolvedValue({
      json: async () => articles,
    } as Response);
    await act(async () => render(<App />, container));

    expect(container.innerHTML).toMatchSnapshot();
  });

  it('should render an error', async () => {
    jest.spyOn(global, 'fetch').mockRejectedValueOnce(new Error('Something went wrong.'));
    await act(async () => render(<App />, container));

    expect(container.querySelector('.alert')?.textContent).toBe('Something went wrong.');
  });
});
