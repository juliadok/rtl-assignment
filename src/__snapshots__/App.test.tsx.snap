// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render articles 1`] = `"<nav class=\\"navbar navbar-expand-sm navbar-dark bg-dark\\"><div class=\\"container\\"><a href=\\"https://www.rtlboulevard.nl/\\" class=\\"text-uppercase font-weight-bold navbar-brand\\">RTL Boulevard</a></div></nav><div class=\\"pt-4 container\\"><div class=\\"row\\"><div class=\\"mb-5 col-md-10 col-12 offset-md-1\\"><p class=\\"text-uppercase text-center\\" data-qa-element=\\"label\\">best gelezen</p><h1 class=\\"text-uppercase text-center font-weight-bold\\" data-qa-element=\\"title\\">De uitschieters van 2018</h1><p class=\\"text-center\\" data-qa-element=\\"description\\">Waar hadden we het dit jaar allemaal over? Je leest het in dit overzicht met de best gelezen artikelen.</p></div></div><a href=\\"https://www.rtlboulevard.nl/entertainment/showbizz/artikel/4245471/trijntje-oosterhuis-onherkenbaar-veel-afgevallen\\" target=\\"_blank\\" class=\\"mb-5 text-decoration-none text-reset row\\"><span class=\\"col-lg-4 col-md-6 col-12\\"><img src=\\"https://www.rtlboulevard.nl/sites/default/files/content/images/2018/06/19/ANP-56919532.jpg\\" data-qa-element=\\"image\\" class=\\"w-100 image\\"></span><span class=\\"col-md-6 col-12\\"><span class=\\"border-0 card\\"><span class=\\"text-uppercase card-title\\" data-qa-element=\\"chapeau\\">Kilo's vliegen eraf</span><span class=\\"mb-2 font-weight-bold card-subtitle\\" data-qa-element=\\"titel\\">Trijntje Oosterhuis onherkenbaar veel afgevallen</span><span class=\\"d-none d-lg-block card-text\\" data-qa-element=\\"lead\\">Onze geliefde souldiva Trijntje Oosterhuis is aanzienlijk veel kilo's verloren! De zangeres plaatst foto's op haar Instagram waar te zien is dat ze een wespentaille heeft waar de meeste vrouwen stront jaloers op zullen zijn. </span></span></span></a><a href=\\"https://www.rtlboulevard.nl/entertainment/tv/artikel/3895726/humberto-tan-over-zijn-vermeende-affaire-met-dionne-stax\\" target=\\"_blank\\" class=\\"mb-5 text-decoration-none text-reset row\\"><span class=\\"col-lg-4 col-md-6 col-12\\"><img src=\\"https://www.rtlboulevard.nl/sites/default/files/content/images/2018/03/28/humbertotan.jpg\\" data-qa-element=\\"image\\" class=\\"w-100 image\\"></span><span class=\\"col-md-6 col-12\\"><span class=\\"border-0 card\\"><span class=\\"text-uppercase card-title\\" data-qa-element=\\"chapeau\\">openhartig</span><span class=\\"mb-2 font-weight-bold card-subtitle\\" data-qa-element=\\"titel\\">Humberto Tan over zijn vermeende affaire met Dionne Stax</span><span class=\\"d-none d-lg-block card-text\\" data-qa-element=\\"lead\\">Humberto Tan neemt woensdagochtend de Giel Mobiel van Veronica-dj Giel Beelen op. Hoog tijd om hem dus de hemd van het lijf te vragen, vindt Giel. </span></span></span></a><a href=\\"https://www.rtlboulevard.nl/entertainment/showbizz/artikel/4237851/patty-brard-totaal-onherkenbaar-na-maagverkleining\\" target=\\"_blank\\" class=\\"mb-5 text-decoration-none text-reset row\\"><span class=\\"col-lg-4 col-md-6 col-12\\"><img src=\\"https://www.rtlboulevard.nl/sites/default/files/content/images/2018/06/14/patty2.jpg\\" data-qa-element=\\"image\\" class=\\"w-100 image\\"></span><span class=\\"col-md-6 col-12\\"><span class=\\"border-0 card\\"><span class=\\"text-uppercase card-title\\" data-qa-element=\\"chapeau\\">strak &amp; stralend</span><span class=\\"mb-2 font-weight-bold card-subtitle\\" data-qa-element=\\"titel\\">Patty Brard totaal onherkenbaar na maagverkleining</span><span class=\\"d-none d-lg-block card-text\\" data-qa-element=\\"lead\\">Het is geen geheim dat Patty Brard een maagverkleining heeft gehad. De diva deelt nu op Instagram een foto waarop te zien is hoe goed ze er nu wel niet uitziet. Pat is strak en stralend!</span></span></span></a><a href=\\"https://www.rtlboulevard.nl/entertainment/tv/artikel/3905846/temptation-tim-heeft-een-schokkende-nieuwe-tattoo\\" target=\\"_blank\\" class=\\"mb-5 text-decoration-none text-reset row\\"><span class=\\"col-lg-4 col-md-6 col-12\\"><img src=\\"https://www.rtlboulevard.nl/sites/default/files/content/images/2018/04/13/Schermafbeelding%202018-04-13%20om%2018.49.13.png\\" data-qa-element=\\"image\\" class=\\"w-100 image\\"></span><span class=\\"col-md-6 col-12\\"><span class=\\"border-0 card\\"><span class=\\"text-uppercase card-title\\" data-qa-element=\\"chapeau\\">Mensen mensen mensen</span><span class=\\"mb-2 font-weight-bold card-subtitle\\" data-qa-element=\\"titel\\">Temptation Tim heeft een schokkende nieuwe tattoo</span><span class=\\"d-none d-lg-block card-text\\" data-qa-element=\\"lead\\">Tim Wauters, beter bekend als Timtation, heeft een nieuwe tattoo laten zetten in de studio van Pommeline en Fabrizio. </span></span></span></a><a href=\\"https://www.rtlboulevard.nl/entertainment/tv/artikel/3891391/gino-en-cherish-gaan-ervandoor-met-verlovingsring-tim\\" target=\\"_blank\\" class=\\"mb-5 text-decoration-none text-reset row\\"><span class=\\"col-lg-4 col-md-6 col-12\\"><img src=\\"https://www.rtlboulevard.nl/sites/default/files/content/images/2018/03/22/Schermafbeelding%202018-03-22%20om%2010.14.13.png\\" data-qa-element=\\"image\\" class=\\"w-100 image\\"></span><span class=\\"col-md-6 col-12\\"><span class=\\"border-0 card\\"><span class=\\"text-uppercase card-title\\" data-qa-element=\\"chapeau\\">Juwelendieven</span><span class=\\"mb-2 font-weight-bold card-subtitle\\" data-qa-element=\\"titel\\">Gino en Cherish gaan ervandoor met verlovingsring Tim</span><span class=\\"d-none d-lg-block card-text\\" data-qa-element=\\"lead\\">Gino en Cherish doen er in Temptation Island alles aan om de relatie van Tim en Deborah om zeep te helpen. En dit lijkt nu te gaan lukken! Vol trots deelt het verleidelijke duo vandaag een kiekje van hun overwinning.</span></span></span></a><a href=\\"https://www.rtlboulevard.nl/entertainment/tv/artikel/3826846/temptations-kevin-ging-niet-5-maar-20-keer-vreemd\\" target=\\"_blank\\" class=\\"mb-5 text-decoration-none text-reset row\\"><span class=\\"col-lg-4 col-md-6 col-12\\"><img src=\\"https://www.rtlboulevard.nl/sites/default/files/content/images/2018/02/16/Schermafbeelding%202018-02-16%20om%2013.58.00_0.png\\" data-qa-element=\\"image\\" class=\\"w-100 image\\"></span><span class=\\"col-md-6 col-12\\"><span class=\\"border-0 card\\"><span class=\\"text-uppercase card-title\\" data-qa-element=\\"chapeau\\">Van 20 walletjes eten</span><span class=\\"mb-2 font-weight-bold card-subtitle\\" data-qa-element=\\"titel\\">Temptations Kevin ging niet 5, maar 20 keer vreemd</span><span class=\\"d-none d-lg-block card-text\\" data-qa-element=\\"lead\\">Joshua kan er niet over uit dat de 18-jarige Megan het toelaat dat haar Kevin, waarmee ze 14 maanden samen is, vijf keer vreemd kan gaan. Nu blijkt het niet om \\"maar\\" vijf keer te gaan, maar zelfs twintig keer!</span></span></span></a><a href=\\"https://www.rtlboulevard.nl/entertainment/tv/artikel/4503981/voormalig-gtst-acteur-dave-mantel-overleden\\" target=\\"_blank\\" class=\\"mb-5 text-decoration-none text-reset row\\"><span class=\\"col-lg-4 col-md-6 col-12\\"><img src=\\"https://www.rtlboulevard.nl/sites/default/files/content/images/2018/12/02/Schermafbeelding%202018-12-02%20om%2015.47.31.png\\" data-qa-element=\\"image\\" class=\\"w-100 image\\"></span><span class=\\"col-md-6 col-12\\"><span class=\\"border-0 card\\"><span class=\\"text-uppercase card-title\\" data-qa-element=\\"chapeau\\">VEEL TE VROEG</span><span class=\\"mb-2 font-weight-bold card-subtitle\\" data-qa-element=\\"titel\\">Voormalig GTST-acteur Dave Mantel overleden</span><span class=\\"d-none d-lg-block card-text\\" data-qa-element=\\"lead\\">Dave Mantel is zaterdagochtend 1 december op 37-jarige leeftijd overleden. Dat meldt zijn directe omgeving via zijn Instagram. De acteur werd bij het grote publiek bekend door de rol van Menno Kuiper in Goede Tijden, Slechte Tijden.</span></span></span></a><a href=\\"https://www.rtlboulevard.nl/entertainment/music/artikel/4124841/andre-hazes-en-monique-uit-elkaar\\" target=\\"_blank\\" class=\\"mb-5 text-decoration-none text-reset row\\"><span class=\\"col-lg-4 col-md-6 col-12\\"><img src=\\"https://www.rtlboulevard.nl/sites/default/files/content/images/2018/04/26/hazesmon.jpg\\" data-qa-element=\\"image\\" class=\\"w-100 image\\"></span><span class=\\"col-md-6 col-12\\"><span class=\\"border-0 card\\"><span class=\\"text-uppercase card-title\\" data-qa-element=\\"chapeau\\">geruchtenmolen</span><span class=\\"mb-2 font-weight-bold card-subtitle\\" data-qa-element=\\"titel\\">'André Hazes en Monique uit elkaar'</span><span class=\\"d-none d-lg-block card-text\\" data-qa-element=\\"lead\\">De geruchten over een eventuele breuk tussen André Hazes en de moeder van zijn kind, Monique, gaan al langer, maar Story laat nu weten dat Hazes nu samen zou wonen met zijn beste vriend en neef Djarno.</span></span></span></a><a href=\\"https://www.rtlboulevard.nl/entertainment/showbizz/artikel/4430091/jeffrey-lake-klapt-uit-de-school-over-onverwachte-breuk-met\\" target=\\"_blank\\" class=\\"mb-5 text-decoration-none text-reset row\\"><span class=\\"col-lg-4 col-md-6 col-12\\"><img src=\\"https://www.rtlboulevard.nl/sites/default/files/content/images/2018/09/26/Schermafbeelding%202018-09-26%20om%2012.01.22.png\\" data-qa-element=\\"image\\" class=\\"w-100 image\\"></span><span class=\\"col-md-6 col-12\\"><span class=\\"border-0 card\\"><span class=\\"text-uppercase card-title\\" data-qa-element=\\"chapeau\\">Ik heb genoeg van al jouw kuren</span><span class=\\"mb-2 font-weight-bold card-subtitle\\" data-qa-element=\\"titel\\">Jeffrey klapt uit de school over onverwachte breuk met Barbie</span><span class=\\"d-none d-lg-block card-text\\" data-qa-element=\\"lead\\">Vorige week werd bekend dat het over en uit is tussen Jeffrey Lake en zijn liefje Samantha de Jong, beter bekend als Barbie. Voor de volkszanger komt de breuk als een donderslag bij heldere hemel, zo vertelt de verbouwereerde Jeff in de nieuwe Story.</span></span></span></a><a href=\\"https://www.rtlboulevard.nl/entertainment/tv/artikel/4209401/no-way-13-reasons-why-mogelijk-van-netflix\\" target=\\"_blank\\" class=\\"mb-5 text-decoration-none text-reset row\\"><span class=\\"col-lg-4 col-md-6 col-12\\"><img src=\\"https://www.rtlboulevard.nl/sites/default/files/content/images/2018/05/28/GettyImages-961398120.jpg\\" data-qa-element=\\"image\\" class=\\"w-100 image\\"></span><span class=\\"col-md-6 col-12\\"><span class=\\"border-0 card\\"><span class=\\"text-uppercase card-title\\" data-qa-element=\\"chapeau\\">slecht nieuws</span><span class=\\"mb-2 font-weight-bold card-subtitle\\" data-qa-element=\\"titel\\">No way: 13 Reasons Why mogelijk van Netflix</span><span class=\\"d-none d-lg-block card-text\\" data-qa-element=\\"lead\\">Het tweede seizoen van de Netflix Original 13 Reasons Why staat pas een dikke week op Netflix, maar toch is er al veel kritiek op de populaire show. Sommige mensen zien zelfs liever dat de show compleet(!) van de buis verdwijnt. </span></span></span></a><a href=\\"https://www.rtlboulevard.nl/entertainment/music/artikel/4131071/details-over-zelfmoord-avicii-bekend\\" target=\\"_blank\\" class=\\"mb-5 text-decoration-none text-reset row\\"><span class=\\"col-lg-4 col-md-6 col-12\\"><img src=\\"https://www.rtlboulevard.nl/sites/default/files/content/images/2018/05/01/Schermafbeelding%202018-04-26%20om%2011.19.31.png\\" data-qa-element=\\"image\\" class=\\"w-100 image\\"></span><span class=\\"col-md-6 col-12\\"><span class=\\"border-0 card\\"><span class=\\"text-uppercase card-title\\" data-qa-element=\\"chapeau\\">Let op: schokkend</span><span class=\\"mb-2 font-weight-bold card-subtitle\\" data-qa-element=\\"titel\\">Details over zelfmoord Avicii bekend</span><span class=\\"d-none d-lg-block card-text\\" data-qa-element=\\"lead\\">TMZ heeft lugubere details over de zelfmoord van Avicii bekendgemaakt. Bronnen hebben dat aan de Amerikaanse website laten weten. </span></span></span></a><a href=\\"https://www.rtlboulevard.nl/entertainment/showbizz/artikel/4303521/barbie-springt-uit-het-raam\\" target=\\"_blank\\" class=\\"mb-5 text-decoration-none text-reset row\\"><span class=\\"col-lg-4 col-md-6 col-12\\"><img src=\\"https://www.rtlboulevard.nl/sites/default/files/content/images/2018/04/26/ANP-28569293.jpg\\" data-qa-element=\\"image\\" class=\\"w-100 image\\"></span><span class=\\"col-md-6 col-12\\"><span class=\\"border-0 card\\"><span class=\\"text-uppercase card-title\\" data-qa-element=\\"chapeau\\">wtf</span><span class=\\"mb-2 font-weight-bold card-subtitle\\" data-qa-element=\\"titel\\">Heftig: 'Barbie uit het raam gesprongen'</span><span class=\\"d-none d-lg-block card-text\\" data-qa-element=\\"lead\\">Volgens Omroep West zou realityster Barbie uit het raam zijn gesprongen. Dat melden verschillende bronnen aan de omroep. </span></span></span></a><a href=\\"https://www.rtlboulevard.nl/entertainment/artikel/4365431/nieuwe-liefde-voor-vader-van-og3ne\\" target=\\"_blank\\" class=\\"mb-5 text-decoration-none text-reset row\\"><span class=\\"col-lg-4 col-md-6 col-12\\"><img src=\\"https://www.rtlboulevard.nl/sites/default/files/content/images/2018/08/12/Schermafbeelding%202018-08-12%20om%2009.40.17_0.png\\" data-qa-element=\\"image\\" class=\\"w-100 image\\"></span><span class=\\"col-md-6 col-12\\"><span class=\\"border-0 card\\"><span class=\\"text-uppercase card-title\\" data-qa-element=\\"chapeau\\">VOL LIEFDE</span><span class=\\"mb-2 font-weight-bold card-subtitle\\" data-qa-element=\\"titel\\">Nieuwe liefde voor vader van OG3NE</span><span class=\\"d-none d-lg-block card-text\\" data-qa-element=\\"lead\\">Ricardo Vol, de vader van Lisa, Amy en Shelley van OG3NE, is opnieuw verliefd. Ruim een jaar nadat hij zijn vrouw Isolde verloor aan kanker, heeft hij weer een relatie.</span></span></span></a><a href=\\"https://www.rtlboulevard.nl/entertainment/tv/artikel/3906371/kritiek-op-zoon-martijn-krabbe\\" target=\\"_blank\\" class=\\"mb-5 text-decoration-none text-reset row\\"><span class=\\"col-lg-4 col-md-6 col-12\\"><img src=\\"https://www.rtlboulevard.nl/sites/default/files/content/images/2018/04/14/Schermafbeelding%202018-04-14%20om%2018.51.58.png\\" data-qa-element=\\"image\\" class=\\"w-100 image\\"></span><span class=\\"col-md-6 col-12\\"><span class=\\"border-0 card\\"><span class=\\"text-uppercase card-title\\" data-qa-element=\\"chapeau\\">vroeg beginnen</span><span class=\\"mb-2 font-weight-bold card-subtitle\\" data-qa-element=\\"titel\\">Kritiek op zoon Martijn Krabbé</span><span class=\\"d-none d-lg-block card-text\\" data-qa-element=\\"lead\\">Er is vrijdagavond en zaterdag veel kritiek op de zoon van Martijn Krabbé. Bickel geniet namelijk volop van een aantal biertjes live op tv bij RTL Late Night. En dat kan niet als 17-jarige, toch?</span></span></span></a><a href=\\"https://www.rtlboulevard.nl/entertainment/showbizz/artikel/4428461/leontine-borsato-houdt-dochter-uit-voice\\" target=\\"_blank\\" class=\\"mb-5 text-decoration-none text-reset row\\"><span class=\\"col-lg-4 col-md-6 col-12\\"><img src=\\"https://www.rtlboulevard.nl/sites/default/files/content/images/2018/09/25/Schermafbeelding%202018-09-25%20om%2009.11.58.png\\" data-qa-element=\\"image\\" class=\\"w-100 image\\"></span><span class=\\"col-md-6 col-12\\"><span class=\\"border-0 card\\"><span class=\\"text-uppercase card-title\\" data-qa-element=\\"chapeau\\">helaas pindakaas</span><span class=\\"mb-2 font-weight-bold card-subtitle\\" data-qa-element=\\"titel\\">Dochter Marco Borsato mag niet meedoen met The Voice</span><span class=\\"d-none d-lg-block card-text\\" data-qa-element=\\"lead\\">Leontine Borsato wil liever niet dat haar dochter Jada met een programma als The Voice of Holland meedoet. Ze vreest dat er anders naar haar gekeken wordt omdat ze 'de dochter van' is.</span></span></span></a><a href=\\"https://www.rtlboulevard.nl/entertainment/showbizz/artikel/4439211/na-maanden-ein-de-lijk-het-geluid-geraden\\" target=\\"_blank\\" class=\\"mb-5 text-decoration-none text-reset row\\"><span class=\\"col-lg-4 col-md-6 col-12\\"><img src=\\"https://www.rtlboulevard.nl/sites/default/files/content/images/2018/10/04/ANP-58356678.jpg\\" data-qa-element=\\"image\\" class=\\"w-100 image\\"></span><span class=\\"col-md-6 col-12\\"><span class=\\"border-0 card\\"><span class=\\"text-uppercase card-title\\" data-qa-element=\\"chapeau\\">het geluid</span><span class=\\"mb-2 font-weight-bold card-subtitle\\" data-qa-element=\\"titel\\">Na maanden is ein-de-lijk Het Geluid geraden</span><span class=\\"d-none d-lg-block card-text\\" data-qa-element=\\"lead\\">De dag van Qmusic-luisteraar Tina Oost uit Tollebeek kan niet meer stuk! De geluksvogel wist het juiste antwoord te geven op Het Geluid en won hiermee maar liefst 58.100 euro.</span></span></span></a><a href=\\"https://www.rtlboulevard.nl/entertainment/showbizz/artikel/4446031/og3ne-doet-boekje-open-over-nieuwe-jonge-vlam-van-vader\\" target=\\"_blank\\" class=\\"mb-5 text-decoration-none text-reset row\\"><span class=\\"col-lg-4 col-md-6 col-12\\"><img src=\\"https://www.rtlboulevard.nl/sites/default/files/content/images/2018/10/10/Schermafbeelding%202018-10-10%20om%2013.38.35.png\\" data-qa-element=\\"image\\" class=\\"w-100 image\\"></span><span class=\\"col-md-6 col-12\\"><span class=\\"border-0 card\\"><span class=\\"text-uppercase card-title\\" data-qa-element=\\"chapeau\\">one happy family</span><span class=\\"mb-2 font-weight-bold card-subtitle\\" data-qa-element=\\"titel\\">OG3NE doet boekje open over nieuwe jonge vlam van vader</span><span class=\\"d-none d-lg-block card-text\\" data-qa-element=\\"lead\\">Rick Vol is al enige tijd dolgelukkig met de 35-jaar jongere én OG3NE-superfan Janine Aarts. En daar zijn dochters Amy, Shelley en Lisa maar wat blij mee. In Privé biechten de zusjes Vol deze week echter op niet veel contact te hebben met de nieuwe vlam van hun papa...</span></span></span></a><a href=\\"https://www.rtlboulevard.nl/entertainment/showbizz/artikel/4489916/rossana-kluivert-houdt-shane-thuis-op-5-december\\" target=\\"_blank\\" class=\\"mb-5 text-decoration-none text-reset row\\"><span class=\\"col-lg-4 col-md-6 col-12\\"><img src=\\"https://www.rtlboulevard.nl/sites/default/files/content/images/2018/11/19/ANP-50072065.jpg\\" data-qa-element=\\"image\\" class=\\"w-100 image\\"></span><span class=\\"col-md-6 col-12\\"><span class=\\"border-0 card\\"><span class=\\"text-uppercase card-title\\" data-qa-element=\\"chapeau\\">oh kom eens kijken</span><span class=\\"mb-2 font-weight-bold card-subtitle\\" data-qa-element=\\"titel\\">Rossana Kluivert houdt Shane thuis op 5 december</span><span class=\\"d-none d-lg-block card-text\\" data-qa-element=\\"lead\\">Rossana Kluivert heeft de afgelopen jaren haar zoon Shane niet naar school laten gaan op 5 december. De reden is de Zwarte Piet-discussie die elk jaar weer oplaait, zo laat ze weten via Instagram. </span></span></span></a><a href=\\"https://www.rtlboulevard.nl/entertainment/tv/artikel/3843461/temptation-tim-ging-met-500-700-vrouwen-naar-bed\\" target=\\"_blank\\" class=\\"mb-5 text-decoration-none text-reset row\\"><span class=\\"col-lg-4 col-md-6 col-12\\"><img src=\\"https://www.rtlboulevard.nl/sites/default/files/content/images/2018/02/23/Schermafbeelding%202018-02-23%20om%2014.53.01.png\\" data-qa-element=\\"image\\" class=\\"w-100 image\\"></span><span class=\\"col-md-6 col-12\\"><span class=\\"border-0 card\\"><span class=\\"text-uppercase card-title\\" data-qa-element=\\"chapeau\\">Op een oude fiets</span><span class=\\"mb-2 font-weight-bold card-subtitle\\" data-qa-element=\\"titel\\">Temptation-Tim ging met 500 à 700 vrouwen naar bed</span><span class=\\"d-none d-lg-block card-text\\" data-qa-element=\\"lead\\">De 30-jarige Tim uit Temptation Island is ook wel beter bekend als de player uit Aalst. Maar dat hij zo'n enorme versierder is heeft niemand zien aankomen. </span></span></span></a><a href=\\"https://www.rtlboulevard.nl/entertainment/showbizz/artikel/4486746/poedelnaakte-yolanthe-flink-onder-vuur\\" target=\\"_blank\\" class=\\"mb-5 text-decoration-none text-reset row\\"><span class=\\"col-lg-4 col-md-6 col-12\\"><img src=\\"https://www.rtlboulevard.nl/sites/default/files/content/images/2018/11/15/Abri-StopOverbevissing-jpg_Page_2.jpg\\" data-qa-element=\\"image\\" class=\\"w-100 image\\"></span><span class=\\"col-md-6 col-12\\"><span class=\\"border-0 card\\"><span class=\\"text-uppercase card-title\\" data-qa-element=\\"chapeau\\">als een vis in het water</span><span class=\\"mb-2 font-weight-bold card-subtitle\\" data-qa-element=\\"titel\\">Poedelnaakte Yolanthe flink onder vuur</span><span class=\\"d-none d-lg-block card-text\\" data-qa-element=\\"lead\\">De FishLove-campagne waarvoor onder andere Tygo Gernandt, Anna Drijver en Yolanthe Sneijder-Cabau voor uit de kleren gingen kan rekenen op flink wat commotie. Vooral Yolanthe heeft het zwaar te verduren...</span></span></span></a><a href=\\"https://www.rtlboulevard.nl/entertainment/tv/artikel/4504816/marian-vond-het-moeilijk-om-naar-boer-zoekt-vrouw-te-kijken\\" target=\\"_blank\\" class=\\"mb-5 text-decoration-none text-reset row\\"><span class=\\"col-lg-4 col-md-6 col-12\\"><img src=\\"https://www.rtlboulevard.nl/sites/default/files/content/images/2018/12/03/Schermafbeelding%202018-12-03%20om%2007.48.41.png\\" data-qa-element=\\"image\\" class=\\"w-100 image\\"></span><span class=\\"col-md-6 col-12\\"><span class=\\"border-0 card\\"><span class=\\"text-uppercase card-title\\" data-qa-element=\\"chapeau\\">Gemengde gevoelens</span><span class=\\"mb-2 font-weight-bold card-subtitle\\" data-qa-element=\\"titel\\">Marian vond het moeilijk om naar Boer zoekt Vrouw te kijken</span><span class=\\"d-none d-lg-block card-text\\" data-qa-element=\\"lead\\">De zoektocht in Boer zoekt Vrouw leverde boer Jaap geen match op. Hij koos voor Marian, maar het is helaas niks tussen de twee geworden. Op de site van het programma laat Marian weten dat ze het moeilijk vond om de beelden van haar en Jaap terug te zien.</span></span></span></a></div>"`;
